src/EnergyPlus/DXCoils.cc

```diff
@@ -16880,7 +16880,7 @@ void CalcVRFCoolingCoil_FluidTCtrl(EnergyPlusData &state,
         }
 
         // If cycling fan, send coil part-load fraction to on/off fan via HVACDataGlobals
-        if (fanOp == HVAC::FanOp::Cycling) state.dataHVACGlobal->OnOffFanPartLoadFraction = PLF;
+        if (fanOp == HVAC::FanOp::Cycling) state.dataHVACGlobal->OnOffFanPartLoadFraction = thisDXCoil.CoolingCoilRuntimeFraction;
```

Comment:
I don't recall why this passes RTF now when it should pass PLF?

Reply:
So that the runtime fraction can be passed into fan simulation here
![image](https://github.com/user-attachments/assets/c3b834d2-4255-43d7-b2d3-b574df604331)

Reply:
I see, you're passing the mass flow rate and RTF for speed 1 and then 0's for massflowrate2 and RTF2 to trick the fan model. But then the fan model also uses state.dataHVACGlobal->OnOffFanPaftLoadFraction to calculate a local RTF. So don't change line 16883, leave it as PLF. Then in the call to the fan, pass the correct RTF. Actually, I don't think this is correct because `thisDXCoil.CoolingCoilRuntimeFraction` will not be the same as `_locRuntimeFrac` (that may not be a bad thing). I think you should pass PLF to the fan (using state.dataHVACGlobal->OnOffFanPartLoadFraction) and then let the fan calculate the _locRuntimeFrac to use for energy calculations, but I'm not actually sure since the coil PLF applies to the load and the fan PLF applies to the air flow. When the coil load and air flow are disconnected like they are in this VRF model, all bets are off as to what is happening.

    Real64 _locRuntimeFrac = (state.dataHVACGlobal->OnOffFanPartLoadFraction >= 1.0)
                                 ? _localFlowFrac
                                 : max(0.0, min(1.0, _localFlowFrac / state.dataHVACGlobal->OnOffFanPartLoadFraction));

Reply:
In the case where the fan is Continuous (the one corresponding to VAV fan in the VRF FluidTCtrl model), this is is how the _locRuntimeFrac is currently computed
```
                if (_useFlowRatiosAndRunTimeFracs) {
                    _locFlowRatio = _localFlowRatio[mode];
                    _locRuntimeFrac = _localRuntimeFrac[mode];
                } else {
                    _locFlowRatio = _localFlowFrac;
                    _locRuntimeFrac = 1.0;
                }
```
and the `_localRuntimeFrac` is calculated like this
```
        if (present(_flowRatio1) && present(_flowRatio2) && present(_runTimeFrac1) && present(_runTimeFrac2)) {
            _useFlowRatiosAndRunTimeFracs = true;
            _localRuntimeFrac[0] = _runTimeFrac1;
            _localRuntimeFrac[1] = _runTimeFrac2;
            _localFlowRatio[0] = _flowRatio1;
            _localAirMassFlow[0] = _localFlowRatio[0] * maxAirMassFlowRate * _localRuntimeFrac[0];
            _localFlowRatio[1] = _flowRatio2;
            _localAirMassFlow[1] = _localFlowRatio[1] * maxAirMassFlowRate * _localRuntimeFrac[1];
        } else {
            _localRuntimeFrac[0] = 1.0; // if runTimeFracs are not present, assume single-mode operation
            _localRuntimeFrac[1] = 0.0; // if runTimeFracs are not present, assume single-mode operation
        }
```
Should I change these to using the calculation like in the Discrete fan case like this?
```
Real64 _locRuntimeFrac = (state.dataHVACGlobal->OnOffFanPartLoadFraction >= 1.0)
                             ? _localFlowFrac
                             : max(0.0, min(1.0, _localFlowFrac / state.dataHVACGlobal->OnOffFanPartLoadFraction));
```
@rraustad

Reply:
I would not change the fan model. I am just pointing out what I see that looks odd. PLF and RTF are an energy adjustment and I am trying to understand how coil PLR/PLF = RTF affect the fan energy. In other cycling coil/system models the fan air flow and coil load are dependent/proportional. In this model they are independent. I think just pass PLF over to the fan and leave it there for now.

Reply:
So I think this VRF coil doesn't have a "Part Load Fraction Correlation Curve Name" field. 
PLF is computed like this
```
        if (thisDXCoil.PLFFPLR(Mode) > 0 && CompCycRatio < 1.0) {
            PLF = CurveValue(state, thisDXCoil.PLFFPLR(Mode), CompCycRatio); // Calculate part-load factor
        } else {
            PLF = 1.0;
        }

        if (PLF < 0.7) {
            if (thisDXCoil.ErrIndex2 == 0) {
                ShowWarningMessage(
                    state,
                    format(
                        "The PLF curve value for the DX cooling coil {} ={:.3R} for part-load ratio ={:.3R}", thisDXCoil.Name, PLF, PartLoadRatio));
                ShowContinueErrorTimeStamp(state, "PLF curve values must be >= 0.7. PLF has been reset to 0.7 and simulation is continuing.");
                ShowContinueError(state, "Check the IO reference manual for PLF curve guidance [Coil:Cooling:DX:SingleSpeed].");
            }
            ShowRecurringWarningErrorAtEnd(
                state, thisDXCoil.Name + ", DX cooling coil PLF curve < 0.7 warning continues...", thisDXCoil.ErrIndex2, PLF, PLF);
            PLF = 0.7;
        }
```
So PLF will constantly be 1. But the coil do cycle. So I feel fan should not have `PLF = 1` be passed into the system fan calculation all the time. Is this right? @rraustad

Reply:
@yujiex @rraustad just pinging here to see if there is still work to do here and try to get this wrapped.